utils.go

```diff
@@ -203,27 +203,27 @@ func getSubidLimits(file string) ([]uint64, error) {
 	}
 	defer f.Close()
 
+	tokens := map[string]uint{
+		"SUB_UID_MIN": 0,
+		"SUB_UID_MAX": 1,
+		"SUB_GID_MIN": 2,
+		"SUB_GID_MAX": 3,
+	}
+
 	scanner := bufio.NewScanner(f)
 	for scanner.Scan() {
 		line := scanner.Text()
-		if strings.Contains(line, "SUB_UID_MIN") {
-			valStr := strings.Split(line, " ")[1]
-			limits[0], err = strconv.ParseUint(valStr, 10, 64)
-		}
-		if strings.Contains(line, "SUB_UID_MAX") {
-			valStr := strings.Split(line, " ")[1]
-			limits[1], err = strconv.ParseUint(valStr, 10, 64)
-		}
-		if strings.Contains(line, "SUB_GID_MIN") {
-			valStr := strings.Split(line, " ")[1]
-			limits[2], err = strconv.ParseUint(valStr, 10, 64)
-		}
-		if strings.Contains(line, "SUB_GID_MAX") {
-			valStr := strings.Split(line, " ")[1]
-			limits[3], err = strconv.ParseUint(valStr, 10, 64)
-		}
-		if err != nil {
-			return limits, fmt.Errorf("failed to parse line %s: %s", line, err)
+		for token, pos := range tokens {
+			if strings.Contains(line, token) {
+				valStr := strings.Fields(line)
+				if len(valStr) < 2 {
```

Comment:
I would change this if-statement to return error if the number of fields are not-equal to "2", that way you are enforcing the expected format as per login.defs(5): 

"The lines consist of a configuration name and value, separated by whitespace."

Reply:
Yes but there may be comments at the end of the line; it's better to leave it as is I think.

Reply:
Not according to the spec:

"Comments are introduced with a "#" pound sign and the pound sign must be the first non-white character of the line."